tutorcairn/plugin.py

```diff
@@ -18,51 +18,53 @@
 
 HERE = os.path.abspath(os.path.dirname(__file__))
 
-
-hooks.Filters.CONFIG_UNIQUE.add_items(
-    [
-        ("CAIRN_CLICKHOUSE_PASSWORD", "{{ 20|random_string }}"),
-        ("CAIRN_POSTGRESQL_PASSWORD", "{{ 20|random_string }}"),
-        ("CAIRN_SUPERSET_SECRET_KEY", "{{ 20|random_string }}"),
-        ("CAIRN_SSO_CLIENT_SECRET", "{{ 20|random_string }}"),
-    ]
-)
-hooks.Filters.CONFIG_DEFAULTS.add_items(
-    [
-        ("CAIRN_VERSION", __version__),
-        ("CAIRN_DOCKER_HOST_SOCK_PATH", "/var/run/docker.sock"),
-        ("CAIRN_HOST", "data.{{ LMS_HOST }}"),
+config: t.Dict[str, t.Dict[str, t.Any]] = {
+    "defaults": {
+        "VERSION": __version__,
+        "DOCKER_HOST_SOCK_PATH": "/var/run/docker.sock",
+        "HOST": "data.{{ LMS_HOST }}",
         # Clickhouse
-        ("CAIRN_RUN_CLICKHOUSE", True),
-        (
-            "CAIRN_CLICKHOUSE_DOCKER_IMAGE",
-            "{{ DOCKER_REGISTRY }}overhangio/cairn-clickhouse:{{ CAIRN_VERSION }}",
-        ),
-        ("CAIRN_CLICKHOUSE_HOST", "cairn-clickhouse"),
-        ("CAIRN_CLICKHOUSE_HTTP_PORT", 8123),
-        ("CAIRN_CLICKHOUSE_HTTP_SCHEME", "http"),
-        ("CAIRN_CLICKHOUSE_PORT", 9000),
-        ("CAIRN_CLICKHOUSE_DATABASE", "openedx"),
-        ("CAIRN_CLICKHOUSE_USERNAME", "openedx"),
+        "RUN_CLICKHOUSE": True,
+        "CLICKHOUSE_DOCKER_IMAGE": "{{ DOCKER_REGISTRY }}overhangio/cairn-clickhouse:{{ CAIRN_VERSION }}",
+        "CLICKHOUSE_HOST": "cairn-clickhouse",
+        "CLICKHOUSE_HTTP_PORT": 8123,
+        "CLICKHOUSE_HTTP_SCHEME": "http",
+        "CLICKHOUSE_PORT": 9000,
+        "CLICKHOUSE_DATABASE": "openedx",
+        "CLICKHOUSE_USERNAME": "openedx",
         # Superset/Postgresql
-        ("CAIRN_RUN_POSTGRESQL", True),
-        ("CAIRN_POSTGRESQL_DATABASE", "superset"),
-        ("CAIRN_POSTGRESQL_USERNAME", "superset"),
-        (
-            "CAIRN_SUPERSET_DOCKER_IMAGE",
-            "{{ DOCKER_REGISTRY }}overhangio/cairn-superset:{{ CAIRN_VERSION }}",
-        ),
-        ("CAIRN_SUPERSET_LANGUAGE_CODE", "{{ LANGUAGE_CODE[:2] }}"),
+        "RUN_POSTGRESQL": True,
+        "POSTGRESQL_DATABASE": "superset",
+        "POSTGRESQL_USERNAME": "superset",
+        "SUPERSET_DOCKER_IMAGE": "{: DOCKER_REGISTRY }}overhangio/cairn-superset:{{ CAIRN_VERSION }}",
```

Comment:
FYI there is a typo that is breaking the build here: `{: DOCKER_REGISTRY }}` -> `{{ DOCKER_REGISTRY }}`
I'm pushing a change now.